### DISCUSSION TITLE
Aitoff Map in Pluto?

### DISCUSSION THREAD

**Discussion by a9lara** [fonsp/Pluto.jl#2180] [Questions about using Pluto]
Hello everyone.

I have been searching and could not find a way to plot an Aitoff map projection in Pluto using plots.jl. I plotted the galactic coordinates, the output is a squared box and not the map projection of the sky.  is there any way to do that. 

Thanks in advance

Cheers
-Armando

**Comment by a9lara** [comment]
Hello?

**Reply by greimel** [reply]
This doesn't sound like a question that it specific to Pluto. You could try https://discourse.julialang.org instead. To increase the chances of being helped, include some runnable code that shows the problem you are facing. (see https://discourse.julialang.org/t/please-read-make-it-easier-to-help-you/14757)

**Reply by a9lara** [reply]
I agree you are right! Thank you and sorry!
